### PR TITLE
Pelican

### PR DESCRIPTION
**Pull request by ocefpaf**
Pelican 3.3 is the last version to work IPython notebooks and octopress theme.

### CODE REVIEW

File: recipes/feedgenerator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+    run:
+        - python
+        - pytz >=0a
```
Comment (pelson) [inline recipes/feedgenerator/meta.yaml]: What kind of version number is that :stuck_out_tongue_closed_eyes:

File: recipes/ghp-import/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    run:
+        - python
+
+test:
+    commands:
+        - ghp-import --help
+about:
+    home: http://github.com/davisp/ghp-import
+    license: Tumbolia Public License
+    summary: Copy your docs directly to the gh-pages branch
```
Comment (pelson) [inline recipes/ghp-import/meta.yaml]: And infinite loop if you put it in CI? :stuck_out_tongue_winking_eye:

**Comment by pelson** [comment]
I'm completely :+1: on these. Nice job @ocefpaf

File: recipes/feedgenerator/meta.yaml (hunk 1 above)
Comment (ocefpaf) [inline recipes/feedgenerator/meta.yaml]: Oops. Thanks for spotting that. (No idea if that was a conda-skeleton bug or I had a mouse/keyboard glitch and edited it without noticing.)

File: recipes/ghp-import/meta.yaml (hunk 2 above)
Comment (ocefpaf) [inline recipes/ghp-import/meta.yaml]: Guilty!

**Comment by ocefpaf** [comment]
Thanks for the review @pelson. Fixed the bad versions and merged.

**Pull request merged**